tutorial/create-a-simple-squid.md

```diff
@@ -140,7 +142,7 @@ npx squid-substrate-metadata-explorer \
 		--out crustVersions.json
 ```
 
-The output is visible in the `crustVersions.json` file, and although the `metadata` field is intelligible, it's worth noting that there are 13 different `versions`, meaning the Runtime has changed 13 times.
+The output is visible in the `crustVersions.json` file, and although the `metadata` field is intelligible, it's worth noting that there are 13 different `versions`, meaning the Runtime has changed 13 (TODO) times.
```

Comment:
Should we mention that the number of versions 13 can be different as it's true for a moment when the article was created. Now it's 18.